app/assets/javascripts/discourse/tests/acceptance/sidebar-plugin-api-test.js

```diff
@@ -1049,5 +1050,23 @@ acceptance("Sidebar - Plugin API", function (needs) {
     assert
       .dom("#d-sidebar .sidebar__panel-switch-button + .sidebar-sections")
       .exists();
+
+    assert
+      .dom(
+        ".sidebar-section[data-section-name='test-chat-channels'] .sidebar-section-header-text"
+      )
+      .doesNotExist();
+
+    withPluginApi(PLUGIN_API_VERSION, (api) => {
+      api.setCombinedSidebarMode();
```

Comment:
The service object is a long lived object so we need to reset its state after each test to ensure that all tests are starting from the same state.

Reply:
good point, on init of service it is cleared